packages/kit/src/hooks/useAppNavigation.ts

```
@@ -68,14 +69,14 @@ import { StackActions } from '@react-navigation/native';
 
 let lastPushAbleNavigation:
   | ReturnType<
-      typeof useNavigation<IPageNavigationProp<any> | IModalNavigationProp<any>>
-    >
+    typeof useNavigation<IPageNavigationProp<any> | IModalNavigationProp<any>>
```

ESLint

[error] Insert `··`

(prettier/prettier)